pom.xml

```diff
@@ -0,0 +1,75 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>io.github.wechaty</groupId>
+  <artifactId>wechaty-grpc-proto</artifactId>
+  <packaging>jar</packaging>
+  <version>1.0.0</version>
+  <name>wechaty-grpc-proto</name>
```

Comment:
Same as the above: `grpc` as the name would be better. (so that our Maven package identifier would be `io.github.wechaty/grpc`?)